Doc/glossary.rst

```diff
@@ -1044,7 +1044,16 @@ Glossary
       as :keyword:`if`, :keyword:`while` or :keyword:`for`.
 
    text encoding
-      A codec which encodes Unicode strings to bytes.
+      A string in Python is a sequence of Unicode code points (in range
+      ``0x0``--``0x10FFFF``). To store or transfer a string, it needs to be
+      serialized as a sequence of bytes.
```

Comment:
Why talking about "serialization" in one way and "recreating [the string]" in the other way? I would prefer to use the terms "encode" and "decode". Here is my definition of an encoding:
https://unicodebook.readthedocs.io/definitions.html#encoding

Reply:
This explanation was basically copied from here (the first paragraph):
https://docs.python.org/3/library/codecs.html#encodings-and-unicode

Julien Palard suggested not to mention internal details (CPU, memory, endianness) and use the word "store or transfer a string", Kyle Stanley fixed some English grammar issues, then it became like this.

It seems the original author wanted to explain the term "encode"/"decode", and tried to avoid using the word "encode"/"decode" in the explanation.

Moreover, maybe this book can be mentioned in `codecs` or `unicodedata` documentation?

Reply:
> It seems the original author wanted to explain the term "encode"/"decode", and tried to avoid using the word "encode"/"decode" in the explanation.

That was my understanding as well. It seems like it would defeat the purpose of having a glossary entry for "text encoding" if the term "encoding" were to be used in the definition itself.